org.aposin.licensescout.core/src/main/java/org/aposin/licensescout/database/DatabaseWriter.java

```diff
@@ -31,7 +31,7 @@
 import org.aposin.licensescout.util.ILFLog;
 
 /**
- * Writes alarms to a database.
+ * Writes build an library data to a database.
```

Comment:
Could you please fix this javaDoc it is not clear to me. (and maybe complete the JavaDocs of the public methods in this class too?) Thanks

Reply:
I extended and completed the documentation.